lib/src/casting_manager.dart

```diff
@@ -53,6 +55,10 @@ class BitmovinCastManager implements BitmovinCastManagerApi {
   );
 
   @override
-  Future<void> updateContext() =>
-      _mainChannel.invokeMethod(Methods.castManagerUpdateContext);
+  Future<void> updateContext() {
+    if (Platform.isIOS) {
+      return Future.value();
+    }
```

Comment:
Would using the async syntax be simpler than handling future explicitly?
```suggestion
  Future<void> updateContext() async {
    if (Platform.isIOS) {
      return;
    }
```

Reply:
I like it: 72cbb5d3